### DISCUSSION TITLE
Switch icon library to Tabler icons

### DISCUSSION THREAD

**Discussion by ShahriarKh** [directus/directus#15301] [Feature Requests]
What do you think about switching the icon library from material to tabler? https://github.com/tabler/tabler-icons
Tabler is frequently updated, is open-source and is beautifully designed, not to mention the number of icons it has.

This is just a personal suggestion, but I'll be really glad to help to implement it if you agree :)

**Comment by benhaynes** [comment]
Not sure about switching, but I'd like to add mapping for vendor-agnostic icon names (like translations) in the app so projects can set their desired library. 👍

**Reply by ShahriarKh** [reply]
that will be really cool

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
